### PR TITLE
MAUI-846428-Prepare UG documentation for ChartInteractive members and Point to value methods

### PR DESCRIPTION
**Pull request by SuryaKaran2143**
**Description**

Included an user guide about the transform axis value to pixel value and vice-versa

### CODE REVIEW

File: MAUI/Cartesian-Charts/Transform-axis-value-to-pixel-value-and-vice-versa.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Transform axis value to pixel value and vice-versa
+
+[`SfCartesianChart`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1%2Ctabid-3%2Ctabid-6%2Ctabid-8%2Ctabid-10%2Ctabid-12%2Ctabid-25%2Ctabid-27%2Ctabid-29%2Ctabid-23%2Ctabid-14%2Ctabid-37%2Ctabid-17%2Ctabid-19%2Ctabid-21) offers two utility methods to transform the pixel to chart point and vice-versa.
+
+* [`ValueToPoint(double value)`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_ValueToPoint_System_Double_) - Converts the data point value to screen point.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Transform-axis-value-to-pixel-value-and-vice-versa.md:14]: API links were wrong, and code snippet was wrong.

Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Transform-axis-value-to-pixel-value-and-vice-versa.md]: I addressed the PR comments
I made the required changes

File: MAUI/Cartesian-Charts/Transform-axis-value-to-pixel-value-and-vice-versa.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# Transform axis value to pixel value and vice-versa
+
+[`SfCartesianChart`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1) offers two utility methods to transform the pixel to chart point and vice-versa.
+
+* [`ValueToPoint(ChartAxis axis,double value)`]() - Converts the data point value to screen point.
+* [`PointToValue(ChartAxis axis,double x, double y)`]() - Converts the screen point to chart value.
+
+{% highlight c# %}
+
+float chartPoint = chart.ValueToPoint(axis, value);
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Transform-axis-value-to-pixel-value-and-vice-versa.md:19]: `protected override void OnTouchDown(ChartBase chart, float pointX, float pointY)
{
    base.OnTouchDown(chart, pointX, pointY);

    if (chart is SfCartesianChart cartesianChart)
    {
        //Convert screen point to chart point.
        var xValue = cartesianChart.PointToValue(cartesianChart.XAxes[0], pointX, pointY);
        var yValue = cartesianChart.PointToValue(cartesianChart.YAxes[0], pointX, pointY);

        //Convert chart point to screen point.
        var xPoint = cartesianChart.ValueToPoint(cartesianChart.XAxes[0], xValue);
        var yPoint = cartesianChart.ValueToPoint(cartesianChart.YAxes[0], yValue);
        
    }
}
`

Comment (Saravanan-Madhesh) [reply MAUI/Cartesian-Charts/Transform-axis-value-to-pixel-value-and-vice-versa.md:19]: Change code snippet.

File: MAUI/Cartesian-Charts/Transform-axis-value-to-pixel-value-and-vice-versa.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+# Transform axis value to pixel value and vice-versa
+
+[`SfCartesianChart`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1) offers two utility methods to transform the pixel to chart point and vice-versa.
+
+* [`ValueToPoint(ChartAxis axis,double value)`]() - Converts the data point value to screen point.
+* [`PointToValue(ChartAxis axis,double x, double y)`]() - Converts the screen point to chart value.
+
+{% highlight c# %}
+
+
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Transform-axis-value-to-pixel-value-and-vice-versa.md:19]: @SuryaKaran2143 , is tab not required for the code snippet?

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

**Review by Saravanan-Madhesh — APPROVED** [review]
The changes are fine.

**Comment by SuryaKaran2143** [comment]
rebuild

**Pull request merged**